### PR TITLE
Add setup wizard command

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## What is the ticket or issue number?
CBAPI-4658

## Pull Request Description
Add a setup wizard to assist in creating a config.toml for the first time

## How Has This Been Tested?
Manual testing as unit tests would be challenging to cover and provide std input

### CODE REVIEW

File: scripts/cbc_syslog_forwarder [hunk 1]
```diff
@@ -19,7 +19,7 @@ import logging.handlers
 import psutil
 import sys
 
-from cbc_syslog import poll, check, history
+from cbc_syslog import poll, check, history, wizard
```
Comment (kebringer-cb) [inline scripts/cbc_syslog_forwarder:22]: Feedback on the history function - apologies that it's late.
* It's not clear what is meant to be provided for "source".  In the wizard it prompts for org name to use in the [header] with url, api id etc.
* Would be nice if the help gave an example timestamp in the format.

File: src/cbc_syslog/core.py [hunk 2]
```diff
@@ -266,3 +270,185 @@ def history(config, start, end, org_key=None):
...
+                minimum_severity = valid_input("Provide a minimum severity between 1 and 10: ",
+                                               lambda x: int(x) > 0 and int(x) <= 10)
+                output_file.write(f"minimum_severity = {minimum_severity}\n")
+
+                print("If you want to provide more rules or add additional filters check out the README for more information\n")
+
+            if input("Do you want to add another organization (y or n): ").lower() == "n":
+                break
+    print(f"""To test your configuration use:
+cbc_syslog_forwarer --log-file /some/path/cbc-syslog.log check {output_file}
```
[the remark this answers is not in the document]
Comment (avanbrunt-cb) [reply src/cbc_syslog/core.py:451]: Oops! I accidentally overwrote the variable

File: src/cbc_syslog/core.py [hunk 3]
```diff
@@ -266,3 +270,185 @@ def history(config, start, end, org_key=None):
...
+        if not func(response):
+            if error_message:
+                print(error_message + "\n")
+            else:
+                print("Invalid Input\n")
+            valid_input(prompt, func, error_message)
+        return response
+
+    with pathlib.Path(output_file).open("w+") as output_file:
+
```
[the remark this answers is not in the document]
Comment (avanbrunt-cb) [reply src/cbc_syslog/core.py:306]: Thanks! I forgot about the base section

File: scripts/cbc_syslog_forwarder (hunk 1 above)
Comment (avanbrunt-cb) [reply scripts/cbc_syslog_forwarder:22]: I am going to correct to source name in the wizard as I don't really want it to be mentally associated with the CBC org name.

Comment (avanbrunt-cb) [reply scripts/cbc_syslog_forwarder:22]: Changed all org name references to source name and provided a sample timestamp

File: src/cbc_syslog/core.py [hunk 4]
```diff
@@ -266,3 +270,186 @@ def history(config, start, end, org_key=None):
...
+        backup_dir = valid_input("Provide an absolute path to an existing backup directory: ",
+                                 lambda resp: pathlib.Path(resp).exists(),
+                                 "Directory not found")
+        output_file.write(f"backup_dir = \"{backup_dir}\"\n")
+
+        output_format = valid_input("What format would you like the data to be sent json or template: ",
+                                    lambda resp: resp.lower() in ["json", "template"]).lower()
+        output_file.write(f"output_format = \"{output_format}\"\n")
+
+        output_type = valid_input("How would you like to data to be sent (udp / tcp / tcp+tls / http / file): ",
```
Comment (emitreva-cb) [inline src/cbc_syslog/core.py:317]: Maybe `the data`? Also maybe unify the messages on 313 and 317 - the options either in () or without

**Review by emitreva-cb — APPROVED** [review]

File: src/cbc_syslog/core.py (hunk 4 above)
Comment (avanbrunt-cb) [reply src/cbc_syslog/core.py:317]: Done

File: scripts/cbc_syslog_forwarder (hunk 1 above)
[the remark this answers is not in the document]
Comment (avanbrunt-cb) [reply scripts/cbc_syslog_forwarder:22]: Added a few sentences to the readme

File: README.md [hunk 5]
```diff
@@ -87,7 +88,11 @@ The `cbc_syslog_forwarder` poll command is designed to be executed in a cronjob
 
 ### Create a Config file
 
-**Coming Soon:** _Wizard setup command to walk through creating a config file from scratch_
+If you are creating a CBC Syslog toml file for the first time checkout the setup wizard which walks you through the basic configuration steps.
+
+    cbc_syslog_forwarer setup my-config.toml
```
Comment (kebringer-cb) [inline README.md:93]: missed a typo

Comment (avanbrunt-cb) [reply README.md:93]: Thanks good catch!

**Review by kebringer-cb — APPROVED** [review]

**Pull request merged**